src/layouts/MainLayout/MainLayout.tsx

```diff
@@ -0,0 +1,23 @@
+import {FC, ReactNode} from 'react';
+
+import {HeaderHome} from '_components/HeaderHome';
+import {AppShell, Group, MediaQuery} from '@mantine/core';
+
+import {useStyles} from './styles';
+
+interface IProps {
+    children: ReactNode;
+}
+
+export const MainLayout: FC<IProps> = ({children}: IProps) => {
+    const {classes} = useStyles();
+    return (
+        <AppShell fixed header={<HeaderHome />}>
+            <MediaQuery largerThan="md" styles={{width: '40vw'}}>
```

Comment:
inline style to class

Reply:
@alekseytsvetkov, styles property applied to child when breakpoint matches, with className propety it doesn't work